### PR TITLE
added ocp-4.11 esx and bm flavor

### PR DESCRIPTION
**Pull request by PariyaAshok**

### CODE REVIEW

File: provision/testdata/flavor_openshift_411_baremetal.inp.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+aci_config:
+  system_id: kube
+  use_legacy_kube_naming_convention: True
```
Comment (snaiksat) [inline provision/testdata/flavor_openshift_411_baremetal.inp.yaml:3]: we seem to copying over an older file; if this is the only input file we are using for testing this flavor, we should not have this configuration set (we could have variations of this file if we also want to test this configuration)

File: provision/testdata/flavor_openshift_411_esx.inp.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+aci_config:
+  system_id: kube
+  use_legacy_kube_naming_convention: True
```
Comment (snaiksat) [inline provision/testdata/flavor_openshift_411_esx.inp.yaml:3]: same comment as earlier

**Review by snaiksat — CHANGES_REQUESTED** [review]
There seems to be something weird about the naming convention for these files:
provision/testdata/flavor_openshift_411_esx_tar/cluster-network-28-ClusterRole-aci-containers:controller.yaml
provision/testdata/flavor_openshift_411_esx_tar/cluster-network-29-ClusterRole-aci-containers:host-agent.yaml
provision/testdata/flavor_openshift_411_esx_tar/cluster-network-29-ClusterRole-aci-containers:host-agent.yaml
provision/testdata/flavor_openshift_411_esx_tar/cluster-network-29-ClusterRole-aci-containers:host-agent.yaml
provision/testdata/flavor_openshift_411_esx_tar/cluster-network-29-ClusterRole-aci-containers:host-agent.yaml

File: provision/testdata/flavor_openshift_411_baremetal.inp.yaml (hunk 1 above)
Comment (PariyaAshok) [reply provision/testdata/flavor_openshift_411_baremetal.inp.yaml]: Okay, config is removed.

**Comment by PariyaAshok** [comment]
Testing in in progress.

**Comment by PariyaAshok** [comment]
Testing completed.

**Comment by snaiksat** [comment]
Fix for filename as follow-up.

**Review by snaiksat — APPROVED** [review]

**Pull request merged**